rules/autoconsent/microsoft.json

```diff
@@ -5,5 +5,5 @@
     "detectPopup": [{ "exists": "#wcpConsentBannerCtrl" }],
     "optOut": [{ "eval": "Array.from(document.querySelectorAll('div > button')).filter(el => el.innerText.match('Reject|Ablehnen'))[0].click() || true" }],
     "optIn": [{ "eval": "Array.from(document.querySelectorAll('div > button')).filter(el => el.innerText.match('Accept|Annehmen'))[0].click() || true" }],
-    "test": [{ "eval": "!!document.cookie.match('MSCC')"}]
+    "test": [{ "eval": "!!document.cookie('MSCC')"}]
```

Comment:
this doesn't look right, you sure you wanted to change this?

Reply:
Sorry and thank your for seeing that - put ".match" back to the rule.